cmd/gossamer/configcmd.go

```diff
@@ -131,21 +135,26 @@ func loadStateAndRuntime(ss *state.StorageState, ks *keystore.Keystore) (*runtim
 
 // getConfig checks for config.toml if --config flag is specified and sets CLI flags
 func getConfig(ctx *cli.Context) (*cfg.Config, error) {
-	fig := cfg.DefaultConfig()
+	defaultConfig := cfg.DefaultConfig()
 	// Load config file.
 	if file := ctx.GlobalString(utils.ConfigFileFlag.Name); file != "" {
-		err := loadConfig(file, fig)
+		configFile := ctx.GlobalString(utils.ConfigFileFlag.Name)
+		log.Debug("Loading config file", "configFile", configFile)
+		err := loadConfig(configFile, defaultConfig)
 		if err != nil {
 			log.Warn("err loading toml file", "err", err.Error())
-			return fig, err
+			return defaultConfig, err
 		}
+	} else {
+		log.Debug("Config File is not set")
 	}
 
 	// Parse CLI flags
-	setGlobalConfig(ctx, &fig.Global)
-	setP2pConfig(ctx, &fig.P2p)
-	setRpcConfig(ctx, &fig.Rpc)
-	return fig, nil
+	log.Debug("Parse CLI flags")
```

Comment:
Doesn't really add any context

Reply:
what do you mean ? you want to remove it ?

Reply:
I don't think this provides any additional information. That is what I'm suggesting

Reply:
I agree. If everything is working as expected and there is no new information, the debug is probably not necessary.